pkgs/development/python-modules/httplib2/default.nix

```diff
@@ -45,12 +39,25 @@ buildPythonPackage rec {
     pytestCheckHook
```

Comment:
When rebasing this ontop of `git merge-base origin/staging origin/master`, I got `ModuleNotFoundError: No module named 'cryptography'` during checkPhase.

Adding `cryptography` to `checkInputs` fixed the build for me.

Reply:
`cryptography` is indeed missing. It's part of `requirements-test.txt` but I missed it. Thanks

`meta` updated to point to https://github.com/httplib2/httplib2

Reply:
I don't see new commits from you, maybe you need to force-push?